tests/unit/test_attribute_mapping.py

```diff
@@ -37,6 +45,14 @@ def test_resource(self):
         expected = self.mappings_json["unifiedAttributeName"]
         self.assertEqual(expected, test.unified_attribute_name)
 
+    @responses.activate
+    def test_delete(self):
+        specific_url = "http://localhost:9100/api/versioned/v1/projects/4/attributeMappings/19629-12"
+        responses.add(responses.DELETE, specific_url, status=204)
+        delete_map = AttributeMapping(self.mappings_json, self.unify)
+        final_response = delete_map.delete
+        self.assertEqual(str(final_response), "<Response [204]>")
```

Comment:
It's safer to do this without converting to a string (since in theory it could be something entirely wrong that happens to have the same repr)--so assert `final_response.status_code` is `204`